### DISCUSSION TITLE
Allow multiple GitHub.com accounts

### DISCUSSION THREAD

**Discussion by ajsirota** [community/community#5684] [Mobile] [answered]
Would be useful for people with separate work and OSS accounts, for example.

**Comment by aconsuegra** [comment, marked as answer]
Hey @ajs256, thanks for you suggestion. I'm adding the `duplicate` label, since we already have similar discussions about this topic, like https://github.com/github/feedback/discussions/3969. Feel free to upvote that one!

**Comment by eliperkins** [comment]
Hey @ajs256, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
